packages/lesmis/src/views/DashboardReportTabView.js

```diff
@@ -49,12 +50,26 @@ const ScrollToTopButton = styled(ArrowUpward)`
 const DEFAULT_DASHBOARD_GROUP = 'Student Enrolment';
 const SCHOOL_DEFAULT_DASHBOARD_GROUP = 'Students';
 
-const getDefaultDashboard = data => {
-  if (!data) {
+// Gets the best default dashboard possible, and check if the selected dashboard is valid
+const useDefaultDashboardTab = (selectedDashboard = null, options) => {
+  const history = useHistory();
+  const { isLoggedIn, isFetching: isFetchingUser } = useUser();
+
+  if (!options) {
```

Comment:
```suggestion
  if (!options || options.length === 0) {
```
Just to match that other change